trojsten/special/plugin_ksp_32_2_1/urls.py

```diff
@@ -2,6 +2,8 @@
 
 from trojsten.special.plugin_ksp_32_2_1 import views
 
+app_name = 'plugin_zwarte'
```

Comment:
nevadi, ze tuna je rovnaky `app_name` ako v `ksp_32_1_1`? nespravi to bordel v url name-och?

Reply:
hm.. pozeram ze ono to vlastne bolo tak uz aj predtm takze asi ok